modules/op-release-notes-1-9.adoc

```diff
@@ -77,7 +77,14 @@ In addition to the fixes and stability improvements, the following sections high
 [id="operator-new-features-1-9_{context}"]
 === Operator
 
-* With this update, you can install {pac} as a separate component, not as a part of `TektonAddon`. You can configure {pac} in the `TektonConfig` CRD.
+* With this update, {pac} is installed by default. You can disable {pac} by using the following flag:
+
++
+[source,yaml]
+
+----
+kubectl patch tektonconfig config --type="merge" -p '{"spec": {"platforms": {"openshift":{"pipelinesAsCode": {"enable": false}}}}}'
```

Comment:
@Dhruv-Soni11 In OpenShift doc, we have to use `oc` command

Reply:
I see everywhere its used `oc` in Openshift doc but ya will defer to @sounix000 input

Reply:
+1 to @jkandasa's comment. Please change any instance of `kubectl` to `oc`

Reply:
@Preeticp - I see from @sounix000's status is OOO till 29th Jan. Can we ask some other writer or should I go ahead and change any instances of `kubectl` to `oc` in this command?

Reply:
Thank you for informing me about this change, Preeti. I have gone ahead and replaced an instance of `kubectl` to `oc` in the content. @VeereshAradhya @ppitonak and @rupalibehera, could you confirm if this command is accurate after this change.